test/services/licences/view-licence-contact-details.service.test.js

```diff
@@ -37,6 +39,16 @@ describe('View Licence Contact Details service', () => {
       postcode: 'CF71 7DG',
       country: 'United Kingdom'
     }])
+    Sinon.stub(FetchCustomerContactDetailsService, 'go').returns([{
```

Comment:
NIT - Love a bit of white space!

```suggestion

    Sinon.stub(FetchCustomerContactDetailsService, 'go').returns([{
```